### DISCUSSION TITLE
Guidelines for creating custom icons

### DISCUSSION THREAD

**Discussion by wendellestradairely** [vmware/clarity#6320] [Ideas]
There should be a guideline in Clarity website for creating and designing custom/non-Clarity icons. In the Angular version of Clarity, it's easier to create custom icons by just pasting the SVG code. But, in the Core version, pasting the code simply doesn't work anymore.

**Comment by bbogdanov** [comment]
Hi @wendellestradairely,

There is documentation how to implement custom icons https://clarity.design/foundation/icons/#customize

[Here](https://github.com/vmware/clarity/blob/7534adfc5840cbd8f827ad3067abc83f563750f0/packages/core/src/card/card.stories.ts#L310) is a code example also where we use custom icons in our storybook for the `Social Post` Card example.

**Reply by mathisscott** [reply]
There is also some info [in this blogpost from a while back](https://medium.com/claritydesignsystem/goodbye-clarity-icons-hello-clarity-core-adb07de9739e#ed12) that might be helpful.
